pgmoon/init.moon

```diff
@@ -32,6 +32,13 @@ flipped = (t) ->
     t[t[key]] = key
   t
 
+int8_serialization = ->
+  -- Lua prior to v5.3 does not support 64-bit integers
+  if 9223372036854775807 - 1 == 9223372036854775807  -- Lua <= 5.2
```

Comment:
I could check the _VERSION variable, but then it's (a) checking strings like "Lua 5.1" and (b) not actually testing the feature in question. With this approach mantissa rounding errors can be caught reliably. If someone compiles Lua with numbers defined as 32-bit, this would break. Then again, quite a few things other things would break as well if that were the case.